src/bin-utils/get-script-by-prefix.test.js

```diff
@@ -0,0 +1,224 @@
+import {isPlainObject, isNull} from 'lodash'
+import getScriptByPrefix, {scriptToObject} from './get-script-by-prefix'
+
+test('scriptToObject returns null when script is not object or string ', () => {
+  const actual = scriptToObject('foo', 42)
+  expect(isNull(actual)).toBeTruthy()
+})
+
+test('scriptToObject returns non-null object when script is string ', () => {
+  const actual = scriptToObject('foo', 'hello')
+  expect(isPlainObject(actual)).toBeTruthy()
+  expect(isNull(actual)).toBeFalsy()
+})
+
+test('scriptToObject returns non-null object when script is string ', () => {
+  const scriptArg = {
+    script: 'hello',
+  }
+  const actual = scriptToObject('foo', scriptArg)
+  expect(isPlainObject(actual)).toBeTruthy()
+  expect(isNull(actual)).toBeFalsy()
+})
+
+test('getScriptByPrefix returns null when no matching script', () => {
+  const config = {scripts: {}}
+  const message = getScriptByPrefix(config, 'w')
+  expect(isNull(message)).toBeTruthy()
```

Comment:
all of these `isNull`s can be: `expect(message).toBeNull()`. We'll get better error messages :smile: